include/wx/webview.h

```diff
@@ -119,6 +119,7 @@ class WXDLLIMPEXP_WEBVIEW wxWebViewFactory : public wxObject
                               const wxSize& size = wxDefaultSize,
                               long style = 0,
                               const wxString& name = wxASCII_STR(wxWebViewNameStr)) = 0;
+    virtual bool IsAvailable() { return true; }
```

Comment:
Should this be documented? I am a bit surprised that `wxWebViewFactory` is documented at all (it could be considered an internal class), but if it is, I think this function should be as well.